sycl/doc/extensions/IntelGPU/IntelGPUDeviceInfo.md

```diff
@@ -0,0 +1,246 @@
+# SYCL(TM) Proposal: New device descriptors for Intel GPUs
```

Comment:
can we not call it all "Intel GPU" extensions, but just Intel's extension for device info (which `SYCL_EXT_INTEL_DEVICE_INFO` feature test macro already says)?

Reply:
Sure!